distribution/docker/src/docker/bin/docker-entrypoint.sh

```diff
@@ -57,15 +57,20 @@ if [[ -f bin/elasticsearch-users ]]; then
   fi
 fi
 
+# Set terminal format according to default ES_LOG_STYLE (and export).
+export CLI_TERMINAL_FORMAT="json"
+
 if [[ -n "$ES_LOG_STYLE" ]]; then
   case "$ES_LOG_STYLE" in
     console)
-      # This is the default. Nothing to do.
+      # This is the default and corresponds to terminal format `json`. Nothing to do.
       ;;
     file)
       # Overwrite the default config with the stack config. Do this as a
       # copy, not a move, in case the container is restarted.
       cp -f /usr/share/elasticsearch/config/log4j2.file.properties /usr/share/elasticsearch/config/log4j2.properties
+      # Clear terminal format to not output json.
+      unset CLI_TERMINAL_FORMAT
```

Comment:
@rjernst Regarding our discussion just now, supporting `ES_LOG_STYLE=file` was why I ultimately decided to pass through some kind of config through the startup scripts.

Reply:
Adding options to docker has support ramifications, it's very difficult to remove these options once added. Additionally, adding anything in the entrypoint should be avoided for the same reasons we moved the cli logic to Java.

For this particular case, isn't ES_LOG_STYLE already exported? In which case, we can read it from the cli.

Reply:
Thanks for the suggestions @rjernst, I think this works out nicely 👍